packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -1370,6 +1381,88 @@ abstract class FlutterCommand extends Command<void> {
     return dartDefineConfigJsonMap;
   }
 
+  /// Parse a property line from an env file.
+  /// Supposed property structure should be:
+  ///   key=value
+  ///
+  /// Where: key is a string without spaces and value is a string.
+  /// Value can also contain '=' char.
+  ///
+  /// Returns a record of key and value as strings.
+  MapEntry<String, String> _parseProperty(String line) {
+    final RegExp blockRegExp = RegExp(r'^\s*([a-zA-Z_]+[a-zA-Z0-9_]*)\s*=\s*"""\s*(.*)$');
```

Comment:
All of these regular expressions are constants. They should be pulled out of this function so that they will not be reallocated on each iteration of the loop in `convertEnvFileToJsonRaw`.